lib/src/audio.dart

```diff
@@ -259,11 +259,11 @@ class AudioPlayer {
   /// [voiceSpeed] Voice speed (1.0 = normal speed)
```

Comment:
Don't these need to be updated for the docs?
`bool transcribe` argument is missing from function args too.
Same update for the docstring in api.dart?